insights/specs.py

```diff
@@ -181,7 +187,23 @@ def docker_container_ids(broker):
 httpd_limits = sf.foreach(httpd_pid, "/bin/cat /proc/%s/limits", name="httpd_limits")
 httpd_ssl_access_log = sf.simple_file("/var/log/httpd/ssl_access_log", name="httpd_ssl_access_log")
 httpd_ssl_error_log = sf.simple_file("/var/log/httpd/ssl_error_log", name="httpd_ssl_error_log")
-httpd_V = sf.simple_command("/usr/sbin/httpd -V", name="http_V", alias="httpd-V")
+
+
+@datasource(ps_auxww)
+def httpd_cmd(broker):
+    ps = broker[ps_auxww].content
+    for p in ps:
+        p_splits = p.split(None, 10)
+        if len(p_splits) >= 11:
+            cmd = p_splits[10].split()[0]
+            # Should compatible with RHEL6
+            # e.g. /usr/sbin/httpd, /usr/sbin/httpd.worker and /usr/sbin/httpd.event
+            if 'httpd' in os.path.basename(cmd):
+                return [cmd]
+    return []
+
+
+httpd_V = sf.foreach(httpd_cmd, "%s -V", name="http_V", alias="httpd-V")
```

Comment:
@xiangce I believe this will need to change now that #831 has been merged.

Reply:
Sure, I changed to foreach_execute, thanks.